Sources/Log.cpp

```diff
@@ -0,0 +1,38 @@
+#include "Log.h"
+
+QFile logs("log-"+ QDateTime::currentDateTime().toString() + "-" + QUuid::createUuid().toString() + ".txt");
+QTextStream logstream(&logs);
+
+void InformationToStream(const QString type, const QMessageLogContext &context, const QString &message, const QString &datetime)
```

Comment:
I think this function is not needed. It is too short, and have too many arguments.